src/Controller/ManualSendController.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+/**
+ * @package Divante\ProductAlert
+ * @author Marek Mularczyk <mmularczyk@divante.pl>
+ * @copyright 2020 Divante Sp. z o.o.
+ * @license See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Divante\ProductAlert\Controller;
+
+use Divante\ProductAlert\Service\ProductAlertService;
+use Shopware\Core\Framework\Routing\Annotation\RouteScope;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * Class ManualSendController
+ *
+ * @RouteScope(scopes={"api"})
+ */
+class ManualSendController extends AbstractController
+{
+    /**
+     * @var ProductAlertService
+     */
+    private $productAlertService;
+
+    /**
+     * ManualSendController constructor.
+     *
+     * @param ProductAlertService $productAlertService
+     */
+    public function __construct(ProductAlertService $productAlertService)
```

Comment:
Ja bym serwisy wstrzykiwał jako Interfejsy. Będzie się to lepiej zgrywać jeżeli ktoś będzie chciał dekorować to później